maint_tools/check_gha_workflow.py

```diff
@@ -127,10 +106,72 @@ def main(args=sys.argv) -> None:
 
     print(df)
 
-    plot_job_durations(df)
+    plot_test_job_durations(df, output_file)
+
+    # %%
+    DOC_WORKFLOW_ID = "37349438"
+    output_file = Path(__file__).parent / "doc_runs_timing.tsv"
+
+    update_tsv(
+        args,
+        update=UPDATE_TSV,
+        output_file=output_file,
+        workflow_id=DOC_WORKFLOW_ID,
+    )
+
+    df = pd.read_csv(
+        output_file,
+        sep="\t",
+        parse_dates=["started_at", "completed_at"],
+    )
+
+    df["duration"] = (df["completed_at"] - df["started_at"]) / pd.Timedelta(
+        minutes=1
+    )
+
+    df = df[df["name"] == "build_docs"]
+    df = df[df["duration"] < 360]
+
+    print(df)
+
+    plot_doc_job_durations(df, output_file)
 
 
-def plot_job_durations(df: pd.DataFrame) -> None:
+def update_tsv(
+    args, update: bool, output_file: Path, workflow_id: str
+) -> None:
+    """Update TSV containing run time of every workflow."""
```

Comment:
If the function is not used elsewhere, should be `_update_tsv`, right ?

Reply:
Good catch. Will fix.

Reply:
done